src/antidote_crdt_big_counter.erl

```diff
@@ -0,0 +1,152 @@
+%% -------------------------------------------------------------------
+%%
+%% Copyright (c) 2014 SyncFree Consortium.  All Rights Reserved.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -------------------------------------------------------------------
+
+%% antidote_crdt_big_counter: A convergent, replicated, operation based Big Counter
```

Comment:
Can you add ~2 lines of documentation? The documentation should at least include something like the warning from Carlos:

> I should add that Big Counter is a named like that because it potentially grows very big, so it should be used with care. The good property is that resets work correctly.